pkg/docker/containers-view.jsx

```diff
@@ -290,7 +291,7 @@ var ImageDetails = React.createClass({
         if (image.Config) {
             entrypoint = image.Config.Entrypoint;
             command = image.Config.Cmd;
-            ports = image.Config.ExposedPorts || [];
+            ports = Object.keys(image.Config.ExposedPorts || {});
```

Comment:
Yes it is. Docker gives us an Object instead of an array there. I've separated this change into its own commit.